include/glow/LLVMIRCodeGen/LLVMBackend.h

```diff
@@ -59,21 +59,21 @@ class LLVMBackendOptions {
 public:
   LLVMBackendOptions();
   /// \returns target triple used by this backend.
-  llvm::StringRef getTarget() const { return target_; }
+  std::string getTarget() const { return target_; }
```

Comment:
In cases, where the result is not supposed to be modified, it is better to return `const std::string &`

Reply:
ok, I will make the changes.